pkgs/tools/graphics/mangohud/combined.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, pkgs, lib
+, libXNVCtrl
+}:
+let
+  mangohud_64 = pkgs.callPackage ./default.nix { inherit libXNVCtrl; };
+  mangohud_32 = pkgs.pkgsi686Linux.callPackage ./default.nix { inherit libXNVCtrl; };
+in
+pkgs.buildEnv rec {
+  name = "mangohud-${mangohud_64.version}";
+
+  paths = [
+    mangohud_32
+  ] ++
+  lib.lists.optionals
+    stdenv.is64bit
+    [ mangohud_64 ]
+  ;
```

Comment:
Could you put these onto a single or two lines? It looks a bit ugly (especially the lone `;`) and we're not even close to 80chars.

Reply:
okay I think I manged to rebase without doing something horrible!